components/cv-form/projects.tsx

```diff
@@ -0,0 +1,298 @@
+                                  >
+                                    <Icons.TrashCan />
+                                  </button>
+                                </div>
+                              ),
+                            )}
+                          <button
+                            className="flex text-indigo-500"
+                            type="button"
+                            onClick={() => arrayHelpers.push("")}
+                          >
+                            <Icons.PlusCircle />
+                            <span>Add</span>
+                          </button>
+                        </div>
+                      )}
+                    />
+                  </div>
+                </div>
+
+                <div className="flex flex-wrap py-8 md:flex-nowrap">
+                  <div className="mb-6 flex flex-shrink-0 flex-col md:mb-0 md:w-64">
+                    <span className="title-font font-semibold text-gray-700 dark:text-gray-400">
+                      Responsibilities
+                    </span>
+                  </div>
+                  <div className="md:flex-grow">
+                    <FieldArray
+                      name={`projects[${projectsIndex}].responsibilities`}
+                      render={(arrayHelpers) => (
+                        <div>
+                          {fProps.values.projects[projectsIndex]
+                            .responsibilities &&
+                            fProps.values.projects[projectsIndex]
+                              .responsibilities.length > 0 &&
+                            fProps.values.projects[
+                              projectsIndex
+                            ].responsibilities.map(
+                              (
+                                responsibility: any,
+                                responsibilitiesIndex: any,
+                              ) => (
+                                <div
+                                  key={responsibilitiesIndex}
+                                  className="mb-2 flex w-full"
+                                >
+                                  <Field
+                                    className="mr-1 w-full rounded-md border border-gray-500 p-1 dark:bg-white"
+                                    name={`projects[${projectsIndex}].responsibilities.${responsibilitiesIndex}`}
+                                  />
+                                  <button
+                                    className="rounded-md border border-indigo-500 bg-indigo-500 p-1  text-white"
+                                    type="button"
+                                    onClick={() =>
+                                      arrayHelpers.remove(responsibilitiesIndex)
+                                    }
+                                  >
+                                    <Icons.TrashCan />
+                                  </button>
+                                </div>
```

Comment:
Responsibility row component.

Reply:
added nested_row reusable component for responsibility array, and also for technologies array